app/controllers/api/v4/converts_controller.rb

```diff
@@ -33,7 +33,8 @@ def create
   private
 
   def check_parameters
-    params.require(:file, :format)
+    params.require(:file)
+    params.require(:format)
```

Comment:
I honestly think something might have changed because I know the test used to pass just fine otherwise I would've fixed it when I added it.